src/main/java/at/hannibal2/skyhanni/features/dungeon/DungeonFinderFeatures.kt

```diff
@@ -156,15 +156,20 @@ class DungeonFinderFeatures {
 
         val stack = event.itemStack
 
+        val classNames = mutableListOf("Healer", "Mage", "Berserk", "Archer", "Tank")
+
         for ((index, line) in stack.getLore().withIndex()) {
             classLevelPattern.matchMatcher(line) {
                 val playerName = group("playerName")
                 val className = group("className")
                 val level = group("level").toInt()
                 val color = getColor(level)
                 event.toolTip[index + 1] = " §b$playerName§f: §e$className $color$level"
+                if(classNames.contains(className)) classNames.remove(className)
             }
         }
+        if(classNames.contains(selectedClass)) classNames[classNames.indexOf(selectedClass)] = "§a${selectedClass}§7"
+        if(config.showMissingClasses) event.toolTip.add("§eMissing: §7" + classNames.joinToString(", "))
```

Comment:
Add another empty line above this new tooltip line. Also change the color of missing to something else to not look too similar to the yellow "click to join" above.